README.md

```diff
@@ -74,14 +80,16 @@ comp.name  # Company Name
 comp.data  # Remaining data as a raw dict.
 ```
 
-Render a dropdown for your user to let them select which of the company files they wish to use. Usually there will only be one against their account, but best to check. Once they've selected, prompt them for the username and password for that company file. Save this as follows:
+Render a dropdown for your user to let them select which of the company files they wish to use. Usually there will only be one against their account, but best to check. If authentication against the company file is needed, prompt them for the username and password for that company file and save this as follows:
```

Comment:
I'd try to be a bit more specific here, as the integration developer might not be entirely familiar with MYOB's options.
Something like this maybe?
```suggestion
Render a dropdown for your user to let them select which of the company files they wish to use. Usually there will only be one against their account, but best to check.
If additional authentication against the company file is needed (ie when the company file account isn't tied via SSO to a my.myob account), prompt them for the username and password for that company file and save this as follows:
```